lib/pages/invitation_selection/invitation_selection.dart

```diff
@@ -98,5 +98,7 @@ class InvitationSelectionController
   }
 
   @override
-  Widget build(BuildContext context) => ContactsSelectionView(this);
+  Widget build(BuildContext context) {
+    return ContactsSelectionDialogView(this);
```

Comment:
Why we need another view for this?

Reply:
I created 2 contact selection view because we use it in new group chat creation and inviting new members to a group, they have different styles it is easier to manage the 2 views like this

Reply:
Because it's not about styles it's about widget structure

Reply:
@sherlockvn I changed my solution, please check again IMO it's cleaner like this

Reply:
Do you update the demo

Reply:
Yes